### PR TITLE
Lemon does pain

### PR DESCRIPTION
**Pull request by LemonInTheDark**
This pr exists so I can test if adding extra plane masters will cause untenable clientside lag.
I will explain what it's doing later, I promise.
Thanks, love you, take care :)

### CODE REVIEW

**Comment by LemonInTheDark** [comment]
Note to self, I accidentially the hard deletes, need to fix that

**Comment by LemonInTheDark** [comment]
Mooshimi also experienced some weird layering with catgirl ears, I think I may have borked some ordering, will need to look into it

**Comment by Mooshimi** [comment]
The pink of cat ears dissapears if you travel z level and then is fixed with a sprite update like toggling mask

**Comment by timothymtorres** [comment]
The blur effects is broken from #68074.  I'm guessing you forgot to add the plane to the list that creates blur effects.
 
I fixed this in an early PR in May by adding a missing plane to the plane masters list.  See: #67191

**Comment by LemonInTheDark** [comment]
> The blur effects is broken from #68074. I'm guessing you forgot to add the plane to the list that creates blur effects.

Nah I refactored plane master controllers and fatfingered a src.plane in there, so it was handing back 70 instances of gameworld

**Comment by LemonInTheDark** [comment]
Fixed the ai thing, and ears not rendering
ai was simple, just eyes not being tracked, the ears was the result of overlay render order yorking, it's regrettably fixed now, but I can't filter so I'm sad

**Comment by LemonInTheDark** [comment]
image
image
This is just lighting not being multiz, and the lower level being totally dark, here's me alphaing the lower lighting plane down a bit, can still see it

File: code/__HELPERS/lighting.dm [hunk 1]
```diff
@@ -1,11 +1,11 @@
 /// Produces a mutable appearance glued to the [EMISSIVE_PLANE] dyed to be the [EMISSIVE_COLOR].
-/proc/emissive_appearance(icon, icon_state = "", atom/offset_spokesman, layer = FLOAT_LAYER, alpha = 255, appearance_flags = NONE)
-	var/mutable_appearance/appearance = mutable_appearance(icon, icon_state, layer, offset_spokesman, EMISSIVE_PLANE, alpha, appearance_flags | EMISSIVE_APPEARANCE_FLAGS)
+/proc/emissive_appearance(icon, icon_state = "", atom/offset_spokesman, layer = FLOAT_LAYER, alpha = 255, appearance_flags = NONE, offset_const)
```
Comment (timothymtorres) [inline code/__HELPERS/lighting.dm:2]: Lemon does this fix #65800

Comment (LemonInTheDark) [reply code/__HELPERS/lighting.dm:2]: yes
I'll explain how when the pr's done

**Comment by LemonInTheDark** [comment]
note to self: the sand bug/persisting appearances is back again, you may have fucked it

**Comment by LemonInTheDark** [comment]
> Paperbins have side effects in `update_overlays()` which gets called in `initialize` due to this PR ...which causes all paperbins to spawn empty

I love this place

**Comment by iprice** [comment]
My linked issues (#68922 and #68923) were closed and I was told to comment them here.  Which the issue report kinda did its self, but I guess I need to note they're not actually resolved as the status implies, thats a consequence of them being closed, as fas as I know they still need resolving and you might as well just refer to the two now closed issues if you need the additional details.

**Comment by LemonInTheDark** [comment]
Thanks. issue jannies like to keep the issue page as JUST current issues on master, with testmerge bugs commented on the particular pr. that's not always possible, but I already had my eye on those, will get to them (alongside my admin pm fuckup) hopefully tomorrow

**Comment by timothymtorres** [comment]
Check and see if this fixes #45669

**Comment by LemonInTheDark** [comment]
It will not fix https://github.com/tgstation/tgstation/issues/45669
https://github.com/tgstation/tgstation/issues/50444 is not intended, and also my fault. We (somehow I still don't know what the actual cause was) fixed it for humans, so it ought to be trivial to fix for slimes

**Pull request closed without merging**